### DISCUSSION TITLE
Can't pair PTM 215Z, device doesn't show up following instructions

### DISCUSSION THREAD

**Discussion by rdumont** [Koenkk/zigbee2mqtt#16761] [Devices]
Hello!

I have a PTM 215Z which I'm trying to pair with my Zigbee2MQTT running on Home Assistant. I follow the [instructions here](https://www.zigbee2mqtt.io/devices/PTM_215Z.html), but the device never appears in the list.

- Zigbee2MQTT version 1.30.1-1
- Mosquitto broker version 6.1.3
- Home Assistant 2023.2.5
- Supervisor 2023.01.1
- Operating System 9.5
- Sonoff_Zigbee_3.0_USB_Dongle_Plus

My Sonoff stick is more than 1 meter away from any router or USB port.

Does anyone have a clue at what could be going on?

**Comment by lux73** [comment]
duplicate https://github.com/Koenkk/zigbee2mqtt/discussions/13398

**Reply by rdumont** [reply]
Thank you for linking! One main difference is that on the linked issue a PTM 215 ZE is being used.

**Reply by lux73** [reply]
ups, sorry just realized they are different devices 🙈

**Comment by rdumont** [comment]
I suppose I missed this part 😞 

> Messages from Green Power devices cannot be "understood" by normal Zigbee devices, therefore they need to be "translated" first. Not all Zigbee devices can do this translation, currently the only devices known to do this are Philips Hue devices.

So this should be the problem.
